src/components/common/StatusBadge.spec.tsx

```diff
@@ -3,33 +3,37 @@ import { Status } from 'shared/types';
 import { renderWithProviders } from 'utils/tests';
 import StatusTag from './StatusTag';
 
-describe('StatusBadge Component', () => {
+describe('StatusTag Component', () => {
```

Comment:
Why change this `StatusBadge.tsx` file to test the `StatusTag` component? Did you mean to create a new test file?